www/pushlink.js

```diff
@@ -44,17 +44,15 @@ var PushLink = function() {
 
 /**
  * Starts PushLink connection
- * @param {string} packageName - The package name for the app
  * @param {string} apiKey - Your PushLink API key
  * @param {string} deviceId - The device id
  * @param {function} successCallback - A function to be called if the command succeeded
  * @param {function} errorCallback - A function to be called if the command failed
  */
-PushLink.prototype.start = function(packageName, apiKey, deviceId, successCallback, errorCallback) {
+PushLink.prototype.start = function(apiKey, deviceId, successCallback, errorCallback) {
```

Comment:
I don't think changing the signature of the public api would be a good idea. We could play with `arguments` or checking the type of `successCallback` (or `deviceId`) to detect whether `packageName` whould be ignored.